src/skin/adminhtml/default/default/omise/omise-dashboard.js

```diff
@@ -0,0 +1,531 @@
+    jQuery.noConflict();
+    jQuery(document).ready(function(){
+
+        // temporary data
+        var chargeData = null, transferData = null;
+
+        // transform charge data into charge table
+        var setChargeTable = function(i, data){
```

Comment:
About this function, I prefer to create `<tr><td>...</td></tr>` inside js file rather than manipulate the DOM. It's about performance and flexible reason.